quarkus-opentelemetry-exporter-gcp/integration-tests/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.opentelemetry.exporter</groupId>
+        <artifactId>quarkus-opentelemetry-exporter-gcp-parent</artifactId>
```

Comment:
We need some integration tests... Is there some mock endpoint or google utility where we can listed to the output?

Reply:
Good point. I will investigate.